### DISCUSSION TITLE
Direct page navigation: Alarm Panel

### DISCUSSION THREAD

**Discussion by jacky-coke** [Blackymas/NSPanel_HA_Blueprint#1763] [Q&A] [answered]
Hi team, 
I'm just working on our alarm system once more and do my best to integrate a smoker friendly version ;-) 

Things start slow with some pre-selected lights turning on and the NSPanel starts playing "The Entertainer" using RTTTL.
After a pre-defined time, things get loud inside our house: sirens, horns and some detectors start screaming ;-) 
 
However, during the first section, I'd like to give myself a chance to disarm the system and would like to show the alarming-panel right away. According to documentation (https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(DE)-HowTo---Alle-wichtigen-Dinge-die-man-wissen-sollte#11-bestimmte-seiten-direkt-aufrufen) I may address the mentioned pages directly. That's great but unfortunately alarm-panel isn't part of the mentioned pages... 

Any chance to address this page directly by automation as well? 

MTIA!! Nik

**Comment by edwardtfn** [comment, marked as answer]
The documentation in German isn't being updated since a while.
The one in English covers that: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/howto.md#entity-specific-pages

**Reply by jacky-coke** [reply]
Uhm, thanks... but where to find the alarm panel? Or is this a case for an entity-specific one?

**Reply by edwardtfn** [reply]
Alarm is an entity-specific.
